tools/create-image.sh

```diff
@@ -26,6 +27,7 @@ display_help() {
     echo
     echo "   -d, --distribution         Set on which debian distribution to create"
     echo "   -f, --feature              Check what packages to install in the image, options are minimal, full"
+    echo "   -s, --size                 Set img size. Currently only supports byte units, default 2048(2G)"
```

Comment:
This is in MB, not "byte units" (in dd with do bs=1M). I would say:
```
Image size (MB), default 2048 (2G)
```

Reply:
Oh, sorry. Thank you for your patience.